src/onegov/fsi/templates/course_event.pt

```diff
@@ -12,26 +12,26 @@
                             <h3 i18n:translate="">Details</h3>
                             <tal:b metal:use-macro="layout.macros['course_event_details']" />
 
-                            <tal:b tal:condition="request.is_logged_in" tal:define="registered model.has_reservation(request.attendee_id)">
+                            <tal:b tal:define="registered model.has_reservation(request.attendee_id)">
                                 <tal:b tal:switch="registered">
                                     <button tal:case="True" class="button success disabled"><i class="fa fa-check" aria-hidden="true"></i> <tal:b i18n:translate>Registered</tal:b></button>
                                     <metal:b tal:case="False" tal:condition="model.bookable" use-macro="layout.macros['intercooler_btn']"/>
                                 </tal:b>
 
                                 <p><strong tal:condition="not: model.bookable" i18n:translate="">This course event can't be booked anymore.</strong></p>
                             </tal:b>
-                            <h3 i18n:translate="">Attendees</h3>
-
-                            <tal:b tal:switch="model.reservations.count() == 0">
-                                <p tal:case="True" i18n:translate>No entries found.</p>
-                                <ul tal:case="False">
-                                <tal:b tal:repeat="reservation model.reservations">
-                                    <li tal:attributes="class python: 'grayed-out' if reservation.is_placeholder else ''">${str(reservation)}</li>
-                                </tal:b>
-                            </ul>
 
+                            <tal:b tal:condition="request.is_admin">
+                                <h3 i18n:translate="">Attendees</h3>
+                                    <tal:b tal:switch="model.reservations.count() == 0">
```

Comment:
`not model.reservations` might work the same, without the cost of `count`, which is slow in Postgres.

Reply:
The problem here with this relationsship is that bool(ResultProxy) is always true, also if there are not results. Is there a less costly way to check the result count?

Reply:
Maybe `.first()` might be a bit faster. But probably not by much. Technically an `exists` query is the quickest way to check this, but those are a bit hard to create in SQLAlchemy. So I'd say leave it as is.